agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/components/Tick.kt

```diff
@@ -44,7 +43,7 @@ fun Tick(
 ) {
     Box(
         modifier = modifier
-            .size(17.5.dp)
+            .size(radius * 2 + strokeWidth)
```

Comment:
I might be wrong, but I assume the stroke is drawn at the middle of the circle outline, so the whole circle would only grow by half of the stroke's width

Reply:
I think that's correct, but because half the stroke's width is added to each side, the overall size increased by the stroke's width. IE top = 1/2 * stroke width and bottom = 1/2 * stroke width, then adding both of these together gets us back to the total stroke width. Using screenshot testing, I can see that if I increase by only half the stroke width then the tick's stroke doesn't look thick enough